routes/app.js

```diff
@@ -39,7 +39,7 @@ module.exports = function () {
         })
     })
 
-    router.get('/admin/usersapi/:securityRating', authHandler.isAuthenticated, appHandler.listUsersAPI)
```

Comment:
This line was the main point of this factor: I really did not like the idea of passing the securityRating like this as a route parameter!

Reply:
After adding Postman tests, I realize it's much easier to leave this route parameter. That way each security rating has its own endpoint to test.